lib/publishing_api_finder_publisher.rb

```diff
@@ -10,7 +10,7 @@ def initialize(finders, logger: Logger.new(STDOUT))
 
   def call
     finders.map do |finder|
-      if finder[:metadata].has_key?("content_id") && !preview_only?(finder)
+      if !preview_only?(finder)
```

Comment:
Could we reverse this so that the positive case comes first? (and the equivalent in the other class...) `if !` is equivalent to `unless` and `unless else` contravenes our [style guide](https://github.com/alphagov/styleguides/blob/master/ruby.md#syntax).

If we'd like to keep the default (not preview-only) case first, we could add another method called something like `publish_everywhere?(finder)` which returns `!preview_only?(finder)` and test that here instead.